gazelle/python/testdata/type_checking_imports_across_packages/a/bar.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
```

Comment:
Nit: copyright 2025 here and elsewhere

Reply:
actually, new files don't need the copyright header anymore, so it would be better to leave it out alltogether.

Reply:
Oh good! When did that change?

Reply:
👍  removed all copyright headers